sapi/fpm/fpm/fpm_unix.c

```diff
@@ -412,7 +416,14 @@ int fpm_unix_init_child(struct fpm_worker_pool_s *wp) /* {{{ */
 	}
 
 #ifdef HAVE_PRCTL
-	if (wp->config->process_dumpable && 0 > prctl(PR_SET_DUMPABLE, 1, 0, 0, 0)) {
+	int dumpable = 1;
```

Comment:
it might be better to initialize it to wp->config->process_dumpable and have shorter condition later.